lib/usd/hdMaya/adapters/materialAdapter.cpp

```diff
@@ -243,6 +198,37 @@ const std::string& HdMayaMaterialAdapter::GetPreviewDisplacementSource() {
     return _PreviewShader().displacementCode;
 }
 
+// Specialized version of :
+// https://en.cppreference.com/w/cpp/algorithm/lower_bound
```

Comment:
What's been specialised about it? _(A comment explaining the changes to the algorithm might be useful in future!)_

Reply:
Good question! I'm actually not terribly familiar with this code. I haven't modified this function at all with this change. I just moved it from higher up in the file to down here closer to its first use so that it was inside the `#if USD_VERSION_NUM <= 1911` conditional. Hopefully it can just go away in the not too distant future if/when we drop support for older versions of USD?

There's more work to be done in this library to get textures working again, so we should revisit this as part of that work.